Jenkinsfile

```diff
@@ -0,0 +1,145 @@
+#!groovy
+//
+// Jenkinsfile to control automated testing using https://jenkins.tjhei.info
+//
+// To run the tests locally, you can execute the following in a burnman working directory:
+//
+// docker run --rm -v "$(pwd):/src" tjhei/burnman:v4 /bin/bash -c "cd /src && ./test.sh"
+
+
+pipeline
+{
+  agent none
+
+  stages
+  {
+
+    stage("check")
+    {
+      agent
+      {
+        docker
+        {
+          image 'dealii/indent'
```

Comment:
dealii/indent?

Reply:
yes, I need some docker image to run this in and this one works. :-)

We might need a different image if we actually want to format code here.